lib/python/picongpu/plugins/sliceFieldReader.py

```diff
@@ -77,7 +77,7 @@ def readFieldSlices(File):
         # go through all valid field vectors
         for x in range(N_x):
             fieldValue = line[x]
-            data[y, x, :] = map(float, fieldValue[1:-1].split(','))
+            data[y, x, :] = [ float(x) for x in fieldValue[1:-1].split(',') ]
```

Comment:
looks good!
Please just [fix the style](https://travis-ci.org/ComputationalRadiationPhysics/picongpu/jobs/438549431) by removing the whitespaces after `[` and before `]` to make the CI happy :)